.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+python:
+  - '2.7'
+
+before_install:
+  - pip install awscli
+  - "./install-apex.sh"
```

Comment:
@PhotoNomad0 Even though the Travis-ci documentation shows it without quotes, shell scripts will not run if they are not in quotes.